interface/patient_file/summary/pnotes_full_add.php

```diff
@@ -266,6 +270,12 @@ function submitform(attr) {
 ?>
    <option value=''><?php echo htmlspecialchars(xl('Mark Note as Completed'), ENT_NOQUOTES); ?></option>
    </select>
+    <?php if ($GLOBALS['messages_feature_date']) { // not in edit screen ?>
+       <b><?php echo htmlspecialchars(xl('Date'), ENT_NOQUOTES); ?>:</b>
+        <?php
+        generate_form_field(array('data_type' => 4, 'field_id' => 'datetime', 'edit_options' => 'F'), empty($datetime) ? date('Y-m-d H:i') : $datetime);
```

Comment:
Are you just doing a standard datetime input field here or something special?

Reply:
Future date and format time - Y-m-d H:i
```
 $('.datetimepicker').datetimepicker({
                <?php $datetimepicker_timepicker = true; ?>
                <?php $datetimepicker_showseconds = false; ?>
                <?php $datetimepicker_formatInput = true; ?>
                <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
                ,minDate : 0 //only future
            })
```

Reply:
Very smart use of generate_form_field for a date/time entry. I never thought to do that :)
Note you'll need to output the proper date format rather than date('Y-m-d H:i'), which can do with:
```
oeFormatDateTime(date('Y-m-d H:i'))
```
(note this is automatically done in generate_form_field() when the entry is not blank)

Reply:
Disregard my comment above. The date('Y-m-d H:i') is being converted by generate_form_field()